ocaml/idl/datamodel.ml

```diff
@@ -954,6 +954,17 @@ module PIF = struct
              Api_errors.pif_has_fcoe_sr_in_use]
       ()
 
+  let pif_set_disallow_unplug = call
```

Comment:
We don't need to prefix this with `pif_` as it's in already the PIF module - this seems to be the convention in this module.